distributions/test/itests/src/test/java/org/codice/ditto/replication/admin/query/ITReplicationQuery.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ditto.replication.admin.query;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.core.Is.is;
+
+import com.jayway.restassured.specification.RequestSpecification;
+import java.net.MalformedURLException;
+import org.junit.Test;
+
+public class ITReplicationQuery {
+
+  private static final String GRAPHQL_ENDPOINT = "https://localhost:8993/admin/hub/graphql";
+
+  private static final String URL = "https://localhost:9999";
+
+  // ----------------------------------- Site Tests -----------------------------------//
+
+  @Test
+  public void createReplicationSite() throws MalformedURLException {
+    String name = "create";
+    asAdmin()
+        .body(makeCreateSiteQuery(name, URL))
+        .when()
+        .post(GRAPHQL_ENDPOINT)
+        .then()
+        .statusCode(200)
```

Comment:
❓ Should we verify the content-type of the responses?

Reply:
Is it really possible to return anything other than json? And, would that be a test of it's own, or an assertion we make in each test?

Reply:
Is there a content-type header in the response? There should be right?